src/sonic-config-engine/minigraph.py

```diff
@@ -103,11 +107,14 @@ def parse_device(device):
             d_type = node.text
         elif node.tag == str(QName(ns, "ClusterName")):
             cluster = node.text
+        elif node.tag == str(QName(ns, "SubType")):
+            d_subtype = node.text
```

Comment:
can we rename this d_subtype, is it supposed to be cable_subtype ?

Reply:
Yes, it is `cable_type` if the device is of `SmartCable`, so I want the attribute name(`SubType`) to be more generic, and in function `get_mux_cable_entries`, it will treat is as cable type (LINE#1754)